### PR TITLE
Help page improvements

### PR DESCRIPTION
**Pull request by mkalininait**
joxi_screenshot_1485949573019

### CODE REVIEW

File: app/assets/images/pages/help/teams.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
+h1 Creating a new team
```
Comment (alicetragedy) [inline app/assets/images/pages/help/teams.txt]: could you remove this file? I think it's not required anymore :)

File: app/views/pages/help.html.slim [hunk 2]
```diff
@@ -1,60 +1,23 @@
...
-  bug, then please [report an issue](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues).
-
-  Or email us at [contact@rgsoc.org](mailto:contact@rgsoc.org)
-
-  Or [tweet](https://twitter.com/RailsGirlsSoC) us :)
+h1 Help
+
+.col-md-4
+  h2 Site and account help
+  ul
```
Comment (alicetragedy) [inline app/views/pages/help.html.slim]: Something minor that doesn't have to be changed right now: the indented links look really weird on the page. This likely has something to do with some css we've added somewhere else for general styling of unordered lists; I'd suggest to (at a later point, when you have time) add a class to the unordered list and override the margin values.

File: app/views/pages/create-an-account.html.slim [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  = image_tag(("pages/help/sign-in-link.png"), class: "help-image")
+
+h2 2. Fill out your profile
+
+p
+  ' Once signed in, you should update your profile information. There are a few mandatory fields:
+  ul
+    li Name
+    li Email
+    li County
```
Comment (alicetragedy) [inline app/views/pages/create-an-account.html.slim]: small typo: change to  “Country”

File: app/views/pages/help.html.slim [hunk 4]
```diff
@@ -1,60 +1,23 @@
...
+   li = link_to "How to create an account", "create-an-account"
+   li = link_to "How to create a team", "create-a-team"
+   li = link_to "How to confirm your coach status", "confirm-a-coach"
+   li = link_to "How to fill out your application", "fill-out-your-application"
+
+.col-md-4
+  h2 Program guides
+  ul
+   li = link_to "Student's Application Guide", "http://railsgirlssummerofcode.org/students/application/"
```
Comment (alicetragedy) [inline app/views/pages/help.html.slim]: could you remove the `'s` here? so just “Student Application Guide”, “Coach Guide”, “Mentor Guide”.

File: app/views/pages/help.html.slim [hunk 5]
```diff
@@ -1,60 +1,23 @@
...
+   li = link_to "Coach's Guide", "http://railsgirlssummerofcode.org/guide/coaching/"
+   li = link_to "Mentor's Guide", "http://railsgirlssummerofcode.org/guide/projects/"
+
+.col-md-4
+  h2 Contact us
+  ul
+    li = link_to "Report an issue on Github", "https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues"
```
Comment (alicetragedy) [inline app/views/pages/help.html.slim]: not sure if I would add a link to the issue tracker here. they will potentially file a lot of random bugs, and I'd prefer that they email us about it first.

File: app/views/pages/help.html.slim [hunk 6]
```diff
@@ -1,60 +1,23 @@
...
+    li = link_to "Write an Email", "mailto:contact@rgsoc.org"
+    li = link_to "Drop a line on Twitter", "https://twitter.com/RailsGirlsSoC"
```
Comment (alicetragedy) [inline app/views/pages/help.html.slim]: change to “Drop us a line on Twitter”

**Comment by alicetragedy** [comment]
Another thing: in your next PR (as I know you're working more on this later, but we want to have a first version up), could you move all the old `application_some_random_title.png` files that are currently in the `app/assets/images/pages` to `app/assets/images/pages/help` which you just created? that way all files are in the same spot. I'm pretty sure those files are just used on the help pages :)

File: app/assets/images/pages/help/teams.txt (hunk 1 above)
Comment (mkalininait) [inline app/assets/images/pages/help/teams.txt]: Oops, you are right, if was a temp file and I committed it accidentally.

File: app/views/pages/help.html.slim (hunk 2 above)
Comment (mkalininait) [inline app/views/pages/help.html.slim]: Do you mean that there are no bullets in the lists? Then yes, I saw it but didn't have time to fix it. It's on my list of things to do.

File: app/views/pages/help.html.slim (hunk 4 above)
Comment (mkalininait) [inline app/views/pages/help.html.slim]: 👍

File: app/views/pages/help.html.slim (hunk 2 above)
Comment (alicetragedy) [inline app/views/pages/help.html.slim]: no, I mean that each link has a very huge margin on the left (I think each item should be left-aligned with the header of each section). regarding the bullet points, they can be added but that's optional I guess. We can think about it later :)

File: app/views/pages/help.html.slim (hunk 5 above)
Comment (mkalininait) [inline app/views/pages/help.html.slim]: It was there in the previous version of the page. I'll remove it.
For "advanced" users, there is a link "Fork me on Github" on the top right corner, which kinda hints that you can submit an issue :)

Comment (alicetragedy) [inline app/views/pages/help.html.slim]: yeah, true :D 
I just remember last year a couple of duplicate issues being submitted; we can always put it back if we see this is a huge problem? what do you think?

**Comment by mkalininait** [comment]
@alicetragedy Those files are not even used anymore. I wanted to delete them. Just need to verify I'm not deleting anything important.

**Comment by alicetragedy** [comment]
@mkalininait awesome! I wasn't sure. You can always delete them in the next iteration of the help page changes if needed then. 👍

File: app/views/pages/help.html.slim (hunk 2 above)
Comment (mkalininait) [inline app/views/pages/help.html.slim]: Would this be better? 
joxi_screenshot_1485958761105

Comment (alicetragedy) [inline app/views/pages/help.html.slim]: definitely looks better! 👍

File: app/views/pages/help.html.slim (hunk 5 above)
Comment (mkalininait) [inline app/views/pages/help.html.slim]: Sure. I agree.

**Comment by mkalininait** [comment]
@alicetragedy Yeah, I'll delete them in the next iteration then. The other changes should be in the latest commit.

**Comment by alicetragedy** [comment]
@mkalininait will take a look and merge as soon as possible. thank you for your great work on this so far <3

**Pull request merged**